src/vuln_analysis/data_models/eval_input.py

```diff
@@ -0,0 +1,76 @@
+from typing import Literal
+
+from pydantic import BaseModel
+from pydantic import Field
+from pydantic import field_validator
+from pydantic import model_validator
+
+from vuln_analysis.data_models.input import ImageInfoInput
+from vuln_analysis.utils.string_utils import is_valid_cve_id
+from vuln_analysis.utils.string_utils import is_valid_ghsa_id
+
+EvalLabel = Literal[
+    "false_positive",
+    "code_not_present",
+    "code_not_reachable",
+    "requires_configuration",
+    "requires_dependency",
+    "requires_environment",
+    "protected_by_compiler",
+    "protected_at_runtime",
+    "protected_at_perimeter",
+    "protected_by_mitigating_control",
+    "uncertain",
+    "vulnerable",
+]
+
+EvalStatus = Literal["AFFECTED", "NOT AFFECTED", "UNKNOWN"]
+
+
+class GroundTruthItem(BaseModel):
+    vuln_id: str
+    status: EvalStatus | None = None
+    label: EvalLabel | None = None
+
+    @field_validator("vuln_id")
+    @classmethod
+    def validate_vuln_id(cls, v: str) -> str:
+        if not (is_valid_cve_id(v) or is_valid_ghsa_id(v)):
+            raise ValueError(f"{v} is not a valid CVE ID or GHSA ID.")
+        return v
+
+
+class ContainerEval(BaseModel):
+    container_image: ImageInfoInput
+    ground_truth: list[GroundTruthItem]
+
+
+class EvalDataset(BaseModel):
```

Comment:
Consider renaming to `AgentMorpheusEvalDataset`?
```suggestion
class AgentMorpheusEvalDataset(BaseModel):
```